### ISSUE TITLE
Generic.Arrays.ArrayIndent Sniff Breaks On Nested Unkeyed Arrays

### ISSUE THREAD

**Issue by rfpludwick** [squizlabs/PHP_CodeSniffer#1980]
The following array declaration is fine according to `Generic.Arrays.ArrayIndent`:

```php
$array = [
    0 => [
        'key' => 'value'
    ]
]
```

The following, however, is marked incorrectly as a violation:

```php
$array = [
    [
        'key' => 'value'
    ]
]
```

**Comment by gsherwood** [comment]
This is a duplicate report of issue #1881, which has been fixed in master and will be released in 3.3.0. Thanks for reporting it though as I was able to confirm this code snippet is also working correctly with the fix.

**Comment by rfpludwick** [comment]
Ah, my mistake Sorry about that!

**Comment by gsherwood** [comment]
> Sorry about that!

Not at all. I'd rather get 2 bug reports than none.
